vlog4j-core/src/test/java/org/semanticweb/vlog4j/core/reasoner/implementation/FileDataSourceTestUtils.java

```diff
@@ -59,9 +60,6 @@
 	public static final String binaryCsvFileNameRoot = "binaryFacts";
 	public static final String invalidFormatNtFileNameRoot = "invalidFormatNtFacts";
 
-	private FileDataSourceTestUtils() {
```

Comment:
I see your point. Originally, I reasoned that since these classes are used within the test packages only, we can make sure to only use them as intended, as no users will come into contact with them. On the other hand, making sure twice and being doubtful of ourselves is probably the safer way of managing the project.